pkg/api/v1/testkube/model_test_suite_execution_extended.go

```diff
@@ -213,3 +213,32 @@ func (e *TestSuiteExecution) IsAborted() bool {
 func (e *TestSuiteExecution) IsTimeout() bool {
 	return *e.Status == TIMEOUT_TestSuiteExecutionStatus
 }
+
+func (e *TestSuiteExecution) convertDots(fn func(string) string) TestSuiteExecution {
```

Comment:
Shouldn't it either:

* Modify the existing execution (as it does) and return its pointer (as it doesn't), or
* Don't modify the existing execution (as it doesn't) and return new object (as it does)
?

Reply:
yes, good point, should be first